### PR TITLE
the EnumConverter  is now able to handle lists of enumeration values

### PR DESCRIPTION
**Pull request by codingWombat**

### CODE REVIEW

**Comment by schroenser** [comment]
I'm only the reviewer when @bannmann opened the pull request.

File: core/src/main/java/com/github/mizool/core/converter/EnumConverter.java [hunk 1]
```diff
@@ -40,4 +45,26 @@ public String fromPojo(Enum<?> enumValue)
         }
         return result;
     }
+
+    public List<String> fromPojos(List<Enum<?>> enumValues)
```
Comment (bannmann) [inline core/src/main/java/com/github/mizool/core/converter/EnumConverter.java]: this is redundant with `com.github.mizool.core.converter.Converters` introduced by PR #92. also, it does not comply with OMG-27 in that it passes `null` collections to the core layer.

File: core/src/main/java/com/github/mizool/core/validation/CheckEnumValue.java [hunk 2]
```diff
@@ -16,10 +16,12 @@
  */
 package com.github.mizool.core.validation;
 
+import java.util.List;
+
 import javax.validation.ConstraintValidator;
 import javax.validation.ConstraintValidatorContext;
 
-public class CheckEnumValue implements ConstraintValidator<EnumValue, String>
+public class CheckEnumValue implements ConstraintValidator<EnumValue, Object>
```
Comment (bannmann) [inline core/src/main/java/com/github/mizool/core/validation/CheckEnumValue.java]: before we adapt this check (and potentially others), we should try using `@EnumValue(...) List<String>` (BeanVal < 2.0) and `List<@EnumValue(...) String>` (as of BeanVal 2.0, using a JDK 8 language change allowing annotations on type usages and not only declarations).

File: core/src/main/java/com/github/mizool/core/converter/EnumConverter.java (hunk 1 above)
Comment (bannmann) [reply core/src/main/java/com/github/mizool/core/converter/EnumConverter.java]: see also https://beanvalidation.org/proposals/BVAL-508/

File: core/src/main/java/com/github/mizool/core/validation/CheckEnumValue.java (hunk 2 above)
Comment (bannmann) [reply core/src/main/java/com/github/mizool/core/validation/CheckEnumValue.java]: see also https://beanvalidation.org/proposals/BVAL-508/

File: core/src/main/java/com/github/mizool/core/converter/EnumConverter.java (hunk 1 above)
Comment (schroenser) [reply core/src/main/java/com/github/mizool/core/converter/EnumConverter.java]: The calls to use the `Converters` class are
`Converters.fromPojos(foos, enumConverter::fromPojo)`
and
`Converters.toPojos(fooDtos, fooDto -> enumConverter.toPojo(Foo.class, fooDto))`
respectively.

**Comment by JanDoerrenhaus** [comment]
@codingWombat Please update this Pull Request, it ages badly.

**Comment by codingWombat** [comment]
as @schroenser statet thare are already calls that cover this scenario

**Pull request closed without merging**